VAMobile/src/screens/BenefitsScreen/Letters/BenefitSummaryServiceVerification/BenefitSummaryServiceVerification.tsx

```diff
@@ -50,24 +43,39 @@ type BenefitSummaryServiceVerificationProps = StackScreenProps<
 function BenefitSummaryServiceVerification({ navigation }: BenefitSummaryServiceVerificationProps) {
   const { t } = useTranslation(NAMESPACE.COMMON)
   const theme = useTheme()
-  const dispatch = useAppDispatch()
-  const { downloading, letterBeneficiaryData, mostRecentServices, letterDownloadError, loadingLetterBeneficiaryData } =
-    useSelector<RootState, LettersState>((state) => state.letters)
+  const { data: letterBeneficiaryData, isLoading: loadingLetterBeneficiaryData } = useLetterBeneficiaryData({
+    enabled: screenContentAllowed('WG_BenefitSummaryServiceVerificationLetter'),
+  })
+  const lettersOptions: LettersDownloadParams = {
+    militaryService: false,
+    serviceConnectedDisabilities: false,
+    serviceConnectedEvaluation: false,
+    nonServiceConnectedPension: letterBeneficiaryData?.benefitInformation.hasNonServiceConnectedPension || false,
+    monthlyAward: false,
+    unemployable: letterBeneficiaryData?.benefitInformation.hasIndividualUnemployabilityGranted || false,
+    specialMonthlyCompensation: letterBeneficiaryData?.benefitInformation.hasSpecialMonthlyCompensation || false,
+    adaptedHousing: letterBeneficiaryData?.benefitInformation.hasAdaptedHousing || false,
+    chapter35Eligibility: false,
+    deathResultOfDisability: letterBeneficiaryData?.benefitInformation.hasDeathResultOfDisability || false,
+    survivorsAward:
+      letterBeneficiaryData?.benefitInformation.hasSurvivorsIndemnityCompensationAward ||
+      letterBeneficiaryData?.benefitInformation.hasSurvivorsPensionAward ||
+      false,
+  }
+  const {
+    isLoading: downloading,
+    isError: letterDownloadError,
+    refetch: refetchLetter,
+  } = useDownloadLetter(LetterTypeConstants.benefitSummary, lettersOptions, { enabled: false })
```

Comment:
Instead of setting this to `false` in each instance of the hook, you can just hard-code it to `false` in the options of the hook itself. Then you wouldn't need to pass `{ enabled: false }` each time you use the hook

Reply:
updated